index.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X#-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mohamed Warsame</title>
+    <link rel="stylesheet" href="./assets/css/styles.css">
+</head>
+<body>
+    <header class="header">
+        <nav class="navbar">
+            <ul>
+                <li><a href="./index.html">Home</a> </li>
+                <li><a href="#about">About</a> </li>
+                <li><a href="#projects">Projects</a> </li>
+                <li><a href="#contact">Contact Me</a> </li>
+            </ul>
+    
+        </nav>
+    </header>
+    <!-- Flex container for nav bar -->
+    <div class="hero">
+      
+
+        <!-- This is my h1 -->
+        <div class="details">
+            <h1>Mohamed Warsame</h1>
+        </div>
+    </div>
+    
+    <!--This is my section for about me  -->
+    <section class="about"
+    id="about">
+        <div><h2>ABOUT ME</h2></div>
+        <div class="main">
+            <div class="profile">
+                <img src="./assets/img/IMG_1343.jpg" alt="profile picture mohamed" class="profile-picture">    
+            </div>
+
+             <div class="intro">
+                    <h3>Here's abit about me</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum incidunt laboriosam recusandae natus commodi illo aliquid labore, fugiat, ullam officiis impedit, tempore tempora. Facilis quia quibusdam unde fugiat maxime magnam.</p>
+            </div>
+             
+            
+        
+        </div>
+    </section>
+    <!--This is my section for featured projects -->
+    <section 
+    id="projects"
+    class="featured-projects">   
+        
+                <h2>Featured Projects</h2>
+            <!-- This is my main featured project and is enlarged -->
+            <div class="box">
+                <div class="card-main">
+                        
+                    <a href="https://mohamedwarsame04.github.io/marketing-agency-refractor-project/" target="_blank"> <img src="./assets/img/placeholder.png" alt="placeholder"></a>
+                    <div class="card-info">
+                        <h4>Horiseon (Marketing Agency Refactor)</h4>  
+                        <p>Lor neque voluptatem veritatis deserunt voluptate sed illum ex inventore sint eius sit sapiente maxime reiciendis?</p>
+                    </div>
+                    
+                </div>
+
+            </div>
+            <!-- These are my other 2 featured projects -->
+            <div class="box">
+
+                    <div class="card">
+                         <a href="https://github.com/MohamedWarsame04"> <img src="./assets/img/placeholder.png" alt="placeholder"></a>
```

Comment:
Replace this placeholder with an image of the actual project as this one is complete.